### PR TITLE
Correct README

### PR DESCRIPTION
**Pull request by radtriste**
**Description of the change:**

**Motivation for the change:**

**Checklist**

- [ ] Add or Modify a unit test for your change
- [ ] Have you verified that tall the tests are passing?

### CODE REVIEW

**Review by spolti — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -36,99 +61,89 @@ make docker-build docker-push IMG=<some-registry>/kogito-serverless-operator:tag
...
+```
 
-**NOTE:** Run `make help` for more information on all potential `make` targets
-
-More information can be found via the [Kubebuilder Documentation](https://book.kubebuilder.io/introduction.html)
-
-# Development and Contributions 
+## Development and Contributions 
 
 Contributing is easy, just take a look at our [contributors](./CONTRIBUTING.md)'guide.
```
Comment (spolti) [inline README.md:149]: do we need to add the license here?

Comment (radtriste) [reply README.md:149]: we have the LICENSE file, so I think indeed we could remove that part

**Review by desmax74 — APPROVED** [review]

**Comment by radtriste** [comment]
waiting for https://github.com/kiegroup/kogito-serverless-operator/pull/41 first

File: README.md (hunk 1 above)
Comment (radtriste) [reply README.md:169]: removed

**Review by davidesalerno — APPROVED** [review]

**Pull request merged**